chainguard-dev/edu
# updating rego doc, and making some more tweaks to discovery onboarding

## Type of change

Minor update to the discovery onboarding, and adding a new section to the rego doc to make mention of custom error and warning messages.

### What should this PR do?

Resolves #348 

### Why are we making this change?


### What are the acceptance criteria? 



### How should this PR be tested?

@amdawson would appreciate a quick sanity check on the Rego update. If it needs more detail LMK and I'll reach out to Hector.
